setup.cfg

```diff
@@ -2,3 +2,159 @@
+# E126 continuation line over-indented for hanging indent
+# E127 continuation line over-indented for visual indent
+# E128 continuation line under-indented for visual indent
+# E129 visually indented line with same indent as next logical line
+# E131 continuation line unaligned for hanging indent
+# E201 whitespace after '('
+# E202 whitespace before ')'
+# E203 whitespace before ':'
+# E211 whitespace before '('
+# E221 multiple spaces before operator
+# E222 multiple spaces after operator
+# E225 missing whitespace around operator
+# E226 missing whitespace around arithmetic operator
+# E227 missing whitespace around bitwise or shift operator
+# E228 missing whitespace around modulo operator
+# E231 missing whitespace after ':'
+# E241 multiple spaces after ','
+# E251 unexpected spaces around keyword / parameter equals
+# E252 missing whitespace around parameter equals
+# E261 at least two spaces before inline comment
+# E262 inline comment should start with '# '
+# E265 block comment should start with '# '
+# E266 too many leading '#' for block comment
+# E271 multiple spaces after keyword
+# E272 multiple spaces before keyword
+# E301 expected 1 blank line
+# E302 expected 2 blank lines
+# E303 too many blank lines
+# E305 expected 2 blank lines after class or function definition
+# E306 expected 1 blank line before a nested definition, found
+# E401 multiple imports on one line
+# E402 module level import not at top of file
+# E501 line too long
+# E502 the backslash is redundant between brackets
+# E701 multiple statements on one line (colon)
+# E702 multiple statements on one line (semicolon)
+# E703 statement ends with a semicolon
+# E704 multiple statements on one line (def)
+# E711 comparison to None should be 'if cond is not None:'
+# E712 comparison to True should be 'if cond is True:' or 'if cond:'
+# E713 test for membership should be 'not in'
+# E714 test for object identity should be 'is not'
+# E721 do not compare types, use 'isinstance()'
+# E722 do not use bare 'except'
+# E731 do not assign a lambda expression, use a def
+# E741 ambiguous variable name
+
+# F401 imported but unused
+# F402 import 'xxx' from line 15 shadowed by loop variable
+# F403 'from xxx import *' used; unable to detect undefined names
+# F405 xxx may be undefined, or defined from star imports: xxx
+# F504 '...' % ... has unused named argument(s)
+# F523 '...'.format(...) has unused arguments at position(s): n
+# F524 '...'.format(...) is missing argument(s) for placeholder(s): n
+# F601 dictionary key 'xs' repeated with different values
+# F821 undefined name
+# F823 local variable 'id' defined as a builtin referenced before assignment
+# F811 redefinition of unused 'xxx' from line 31
+# F841 local variable name is assigned to but never used
+
+# W191 indentation contains tabs
+# W503 line break before binary operator
+# W504 line break after binary operator
+# W605 invalid escape sequence '\d'
+
+ignore =
+    E101,
```

Comment:
Can the comments go inline with the error code here? I worry that if these change the comments aren't going to get updated in the future

Reply:
You mean putting the comment above the ignored code?

Reply:
Was thinking maybe something like:

``` python
W605, # invalid escape sequence '\d'
```

Reply:
That would work